app/main.py

```diff
@@ -2,0 +1,46 @@
+from __future__ import annotations
+
+
+class Animal:
+    def __init__(
+            self,
+            name: str,
+            appetite: int,
+            is_hungry: bool = True
+    ) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
```

Comment:
This `print_name` method could be a `@staticmethod` according to the checklist. However, since it uses the `self` keyword to access the instance's name, it cannot be a static method. A static method does not have access to any instance or class-specific data.